python/tests/operations/slice.py

```diff
@@ -0,0 +1,696 @@
+import os
+import unittest
+
+import numpy as np
+
+import equistore.io
+from equistore.operations import slice, remove_gradients
+from equistore import Labels, TensorBlock, TensorMap
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
+TEST_FILE = "qm7-spherical-expansion.npz"
+
+
+class TestSlice(unittest.TestCase):
```

Comment:
Is there a test in which you are slicing both on samples and properties?

Reply:
Yes it does, see test block 5. In docstring on TestSlice: "5) Slicing samples and properties dimensions simultaneously."